docs/source/_templates/docs-sidebar.html

```diff
@@ -1,7 +1,4 @@
 
-<a class="navbar-brand" href="{{ pathto(master_doc) }}">
```

Comment:
Converted the PR to a Draft one because of this change.
See https://github.com/apache/arrow-datafusion/issues/5500#issuecomment-1473877086

Reply:
(I personally think that as long as this removal doesn't render the docs inoperable,  it would be fine to remove this code to get the docs publishing and fix it afterwards)